docs/user/metal/install_ipi.md

```diff
@@ -177,6 +177,65 @@ pullSecret: ...
 sshKey: ...
 ```
 
+#### Required Inputs
+
+| Parameter | Default | Description |
+| --- | --- | --- |
+`provisioningNetworkInterface` | | The name of the network interface on control plane nodes connected to the provisioning network. |
+`hosts` | | Details about bare metal hosts to use to build the cluster. See below for more details. |
+`defaultMachinePlatform` | | The default configuration used for machine pools without a platform configuration. |
+`apiVIP` | `api.<clusterdomain>` | The VIP to use for internal API communication. |
+`ingressVIP` | `test.apps.<clusterdomain>` | The VIP to use for ingress traffic. |
+`dnsVIP` | | The VIP to use for internal DNS communication. |
+
+##### VIP Settings
+
+The `apiVIP` and `ingressVIP` settings must either be provided or
+pre-configured in DNS so that the default names resolve correctly.
```

Comment:
Should we say what those names are?

Reply:
I added a reference to the table in the previous section that gives the names.

Reply:
Ah, I glossed over that. Thanks.